examples/enums/enums.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/io;
+
+// Enums are defined using `enum` modifier.
```

Comment:
Is it correct to call `enum` a modifier? Usually we use that term for keywords which modifies the semantics of the construct. e.g., `resource`, `remote`, `public`, `private`.

Reply:
Yeah this didn't read right to me also.